assets/company-admin/components/CompanyDetailsProductRow.jsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 
 import {gettext} from 'utils';
-
+import {getConfig} from 'utils';
```

Comment:
Please place all imports from the same file in one import statement

Reply:
my bad I forgot to fix this.

Reply:
This one too: https://github.com/superdesk/newsroom-core/pull/376/files#diff-f09ede58af181c8ed54d9e146236debe1d85a168995f4a22422c8e2acb19d9e3R10